src/app_options.cpp

```diff
@@ -289,6 +295,10 @@ Input_platform App_options::input_platform() const noexcept
    return _input_platform;
 }
 
+std::string App_options::user_string_dict() const noexcept
+{
+   return _user_string_dict;
+}
```

Comment:
```suggestion
}

```
Very minor style thing, looks like a newline is missing in between the two functions.